index.html

```diff
@@ -454,6 +460,140 @@ <h2 class="h2">Grid</h2>
+              <i class="fas fa-times close"></i>
+            </div>
+        </code></pre>
+     
+    <div class="snackbar snackbar-secondary my-1">
+      <i class="fas fa-info-circle"></i>
+      <span
+        >Lorem ipsum dolor sit amet consectetur adipisicing elit. Asperiores
+        explicabo iusto earum, quos nam ut?</span
+      >
+      <i class="fas fa-times close"></i>
+    </div>
+
+    <div class="snackbar snackbar-primary my-1">
+      <i class="fas fa-info-circle"></i>
+      <span>Lorem ipsum dolor sit amet consectetur.</span>
+      <i class="fas fa-times close"></i>
+    </div>
+
+    <div class="snackbar snackbar-success my-1 snackbar-bottom-right">
+      <i class="fas fa-info-circle"></i>
+      <span>Lorem ipsum dolor sit amet consectetur.</span>
+      <i class="fas fa-times close"></i>
+    </div>
+      </section>
+      <section id="navar">
+        <h2 class="h2">Navbar</h2>
+        
+        <div class="navbar">
+      <div class="nav-left"><i class="fas fa-home"></i>Home</div>
+      <div class="nav-right">
+        <div class="input-wrapper w-icon">
+          <i class="fas fa fa-search"></i>
+          <input type="text" class="input" />
+        </div>
+        <i class="far fa-heart"></i>
+        <i class="far fa-bell"></i>
+        <i class="fas fa-cart-plus"></i>
+        <div class="btn btn-link btn-link-secondary">Logout</div>
+      </div>
+    </div>
+
+    <div class="navbar my-1 navbar-dark">
+      <div class="nav-left"><i class="fas fa-home"></i>Home</div>
+      <div class="nav-right">
+        <div class="input-wrapper w-icon">
+          <i class="fas fa fa-search"></i>
+          <input type="text" class="input" />
+        </div>
+        <i class="far fa-heart"></i>
+        <i class="far fa-bell"></i>
+        <i class="fas fa-cart-plus"></i>
+        <div class="btn btn-link btn-link-secondary">Logout</div>
+      </div>
+    </div>
+    <div class="navbar my-1 navbar-primary">
+      <div class="nav-left"><i class="fas fa-home"></i>Home</div>
+      <div class="nav-right">
+        <div class="input-wrapper w-icon">
+          <i class="fas fa fa-search"></i>
+          <input type="text" class="input" />
+        </div>
+        <i class="far fa-heart"></i>
+        <i class="far fa-bell"></i>
+        <i class="fas fa-cart-plus"></i>
+        <div class="btn btn-link btn-link-secondary">Logout</div>
+      </div>
+    </div>
+
+    <pre><code class="langauge-html">
+      <div class="navbar">
+      <div class="nav-left"><i class="fas fa-home"></i>Home</div>
+      <div class="nav-right">
+        <div class="input-wrapper w-icon">
+          <i class="fas fa fa-search"></i>
+          <input type="text" class="input" />
+        </div>
+        <i class="far fa-heart"></i>
+        <i class="far fa-bell"></i>
+        <i class="fas fa-cart-plus"></i>
+        <div class="btn btn-link btn-link-secondary">Logout</div>
+      </div>
+    </div>
+
+    <div class="navbar my-1 navbar-dark">
```

Comment:
As It is a navbar, you can use the `<nav>` semantic tag here, I think, it will help in more reachability.